pkgs/by-name/ll/llama-cpp/package.nix

```diff
@@ -105,6 +105,12 @@ stdenv.mkDerivation (finalAttrs: {
     "-DCMAKE_C_COMPILER=hipcc"
     "-DCMAKE_CXX_COMPILER=hipcc"
     "-DCMAKE_POSITION_INDEPENDENT_CODE=ON"
+
+    # Build all targets supported by rocBLAS. When updating search for TARGET_LIST_ROCM
```

Comment:
This code snippet was lifted from the `llama.cpp` repo:

- https://github.com/ggerganov/llama.cpp/blob/e39106c0554cbd0e9310e08fb3b2a577ea4b6273/.devops/nix/package.nix#L185C9-L189C166